doc/manuals/admin/build_source.md

```diff
@@ -53,7 +53,7 @@ commands that require root privilege):
 
 * Install Google Test/Mock from sources (there are RPM pacakges for this, but they don't seem to be working with the current CMakeLists.txt configuration). Previously the URL was http://googlemock.googlecode.com/files/gmock-1.5.0.tar.bz2 but google removed that package in late August 2016 and it is no longer working.
```

Comment:
pacakges  -> packages

(This typo hasn't been introduced by the PR but given you are touching text near it, it is a good opportunity to fix :)

Reply:
ok)

Reply:
fixed in https://github.com/telefonicaid/fiware-orion/pull/3117/commits/ce41e406315343b6c61358ad41eda2854b4a1b63